docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<p>Your math is better than mine. Can you educate me on what the <code>triangle over equal sign</code> means?</p><p> </p><p>Also, why are we calling it <code>X_t</code> ? Why <code>X</code>?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
> what the `triangle over equal sign` means?

if I am not mistaken, it means "define/call" something. I used the same symbol when I was writing eq(6) of the notebook. So, if we want to define a term ourselves, we use this symbol. I think it is somewhat similar to assignment operator `:=`. I did this to distinguish between the name of variables in the problem, and the names I created/defined myself. 

> Also, why are we calling it `X_t` ? Why `X`?

Did you mean "why NOT `X`?" ?, or Did you mean why the letter `X`? I used the subscript `t` to show that this variable changes with `t`. Why `X`? No reason. Now that you asked, I think I should have used a better name such as `d_t` as the distance between two corresponding elements  at time stamp `t`.

Reply:
So, is it okay to keep the name `X_t` (for now)?

Reply:
I propose a more neutral variable like the `\alpha_t` or `\beta_t`

Reply:
Sounds good! I will revise notebook accordingly in the next set of commits.